checker/tests/nullness/TypeVarPrimitives.java

```diff
@@ -0,0 +1,26 @@
+// Unannotated version in framework/tests/all-systems/Primitives.java
```

Comment:
Should this be `framework/tests/all-systems/TypeVarPrimitives.java`? At least that's the file being added below.